### DISCUSSION TITLE
are SH script and Legit host spoofing detected by others?

### DISCUSSION THREAD

**Discussion by snowleopardd** [YimMenu/YimMenu#3383] [Q&A]
just wondering if these 2 options can be detected by other modders

**Comment by ghost** [comment]
> just wondering if these 2 options can be detected by other modders

As far as I know, we can track the change of script hosts, but we need to take into account many points, even the banal ones that the script host can be given to another player.  It is impossible to say for sure whether the host script is a modder or not.

As for the legit token, it seems that we have fixed it and 2t1 no longer detects it.  or at least for now.

**Reply by lonelybud** [reply]
both are detectable. One can detect `force script host` if another player is not willing to give it to you on your request as per theory. Legit host token spoofing from theory spoof's token in a given range which is detectable by other menus possibly but not yim.

**Reply by snowleopardd** [reply]
> both are detectable. One can detect `force script host` if another player is not willing to give it to you on your request as per theory. Legit host token spoofing from theory spoof's token in a given range which is detectable by other menus possibly but not yim.

so basically the legit host token is useless
